docs/docs/Concepts/concepts-flows.md

```diff
@@ -10,6 +10,135 @@ If you've already got a Langflow JSON file, import it into Langflow by clicking
 
 Once imported, your flow is ready to use.
 
+:::tip
+You can drag and drop Langflow JSON files directly from your file system into the Langflow window to import a flow, even into the initial Langflow splash screen.
+:::
+
 ## Export Flow
 
 The option to export a flow is available in the same menu as shown above. Once exported as JSON, you can import your flow into another Langflow instance.
+
+## Langflow JSON file contents
+
+Langflow JSON files contain **nodes**, **edges**, and **additional information** that describe the flow's components and settings.
```

Comment:
Suggest adding links to the sections for nodes, edges, and additional information.